test/library/standard/Path/verma-varsha/test_expandVars.chpl

```diff
@@ -0,0 +1,7 @@
+use Path;
+
+var x1:string = "$CHPL_HOME/bin/${CHPL_HOST_PLATFORM}";
+assert(expandVars(x1) == CHPL_HOME+"/bin/"+CHPL_HOST_PLATFORM);
+
+var x2:string = "bin/$CHPL_COMM/$CHPL_COMM_SUBSTRATE/";
+assert(expandVars(x2) == "bin/"+CHPL_COMM+"/"+CHPL_COMM_SUBSTRATE+"/");
```

Comment:
This assert fails when I run it.

Reply:
I don't know what's going wrong. It's passing all the tests on the Travis CI as well as on my machine. :(

Reply:
Ah, I see.  I hadn't noticed that your most recent commit not only added the test, but also addressed the bug in the function.  I was running it with the old version of the function.

It works for me as well with the new version of the function.